modules/nextflow/src/main/groovy/nextflow/processor/TaskProcessor.groovy

```diff
@@ -1781,7 +1786,7 @@ class TaskProcessor {
         if( obj instanceof Path )
             return obj
 
-        if( obj == null )
+        if( !obj == null )
```

Comment:
Is this not a bug?

Reply:
Indeed. I wanted to make a separate PR with a test but then I forgot about that